test/libsolidity/SolidityParser.cpp

```diff
@@ -112,52 +112,6 @@ while(0)
 
 BOOST_AUTO_TEST_SUITE(SolidityParser)
 
-BOOST_AUTO_TEST_CASE(multiple_return_param_trailing_comma)
-{
-	char const* text = R"(
-		contract test {
-			function() returns (uint a, uint b,) {}
-		}
-	)";
-	CHECK_PARSE_ERROR(text, "Unexpected trailing comma in parameter list.");
-}
-
-BOOST_AUTO_TEST_CASE(multiple_modifier_arg_trailing_comma)
-{
-	char const* text = R"(
-		contract test {
-			modifier modTest(uint a, uint b,) { _; }
-			function(uint a) {}
-		}
-	)";
-	CHECK_PARSE_ERROR(text, "Unexpected trailing comma in parameter list.");
-}
-
-BOOST_AUTO_TEST_CASE(multiple_event_arg_trailing_comma)
-{
-	char const* text = R"(
-		contract test {
-			event Test(uint a, uint b,);
-			function(uint a) {}
-		}
-	)";
-	CHECK_PARSE_ERROR(text, "Unexpected trailing comma in parameter list.");
-}
-
-BOOST_AUTO_TEST_CASE(two_exact_functions)
-{
-	char const* text = R"(
-		contract test {
-			function fun(uint a) returns(uint r) { return a; }
-			function fun(uint a) returns(uint r) { return a; }
-		}
-	)";
-	// with support of overloaded functions, during parsing,
```

Comment:
This should not be removed.

Reply:
ok right, I actually didn't read the comment, just saw that it was not retained :)